File: Typescript/Practice/medium/medium_burglarySeries.py
#!/usr/bin/env python

def add_name(dict1, str1, num):
  dict1[str1] = num
  return dict1

print(add_name({}, "Brutus", 300))
print(add_name({ "piano": 500 }, "Brutus", 400))
print(add_name({ "piano": 500, "stereo": 300 }, "Caligula", 440))

File: Typescript/Practice/medium/medium_burglarySeries.ts
#!/usr/bin/env npx ts-node

function addName (obj: {[key: string]: number}, str: string, num: number): {[key: string]: number} {
  obj[str] = num;
  return obj;
};

console.log(addName({}, "Brutus", 300));
console.log(addName({ piano: 500 }, "Brutus", 400));
console.log(addName({ piano: 500, stereo: 300 }, "Caligula", 440));
